apps/shade/src/lib/utils.ts

```
@@ -549,11 +549,11 @@ export const formatDisplayDateWithRange = (date: string, range: number): string
 
 // Helper function to format member names with fallback to email
 export const formatMemberName = (member: {name?: string; email?: string}) => {
-    return member.name || member.email || 'Unknown Member';
+    return (member.name && member.name.trim()) || member.email || 'Unknown Member';
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)